### DISCUSSION TITLE
Question about metagenomic assembly

### DISCUSSION THREAD

**Discussion by sf202303** [ablab/spades#1118] [Q&A]
Hi everyone, I am working on the metagenomic assembly for Illumina 2x250 reads. I found "We suggest using 350-500 bp fragments with 2x150 reads and 550-700 bp fragments with 2x250 reads." (in the section "Assembling long Illumina paired reads (2x150 and 2x250)"). Does that mean I need to trim my reads first and only keep those with 550-700 bp? If so, any recommended trimming tool? Thank you very much.

**Comment by asl** [comment]
Hello

The suggestion is about *fragments* not reads. Fragment length distribution is a feature of your sequencing library and really depends on how it was prepared. You may want to talk to your sequencing center about fragment length distribution (aka insert size distribution).

**Reply by sf202303** [reply]
Hi asl, 

Thanks for your prompt response. I did the miseq Illumina 2*250bp myself. For the fragment length distribution, are you referring to the fragment size measured by the bioanalyzer?  
If so, the average size of the pooled library was 916bp, and the size distribution in CV was 92.9%. The individual samples ranged from 735bp-1074bp, and the size distribution in CV ranged from 79.7%  to 94.9%. Would this be an issue?
If not, where should I look for the fragment length distribution? Thank you!

**Reply by sf202303** [reply]
Hi Anton,

Could you help with my follow-up questions? Thank you very much.

**Reply by asl** [reply]
Ok, so just proceed directly to the assembly

**Reply by sf202303** [reply]
Thanks for your reply! So the bioanalyzer fragment size could be higher than 700bp? What fragment range may not be preferred?

**Reply by asl** [reply]
There is no "bad range". Fragment length distribution effectively influences which repeats might be span by these fragments and what you can do around this. E.g. if your left and right reads overlap, then you could resolve only shorter repeats, etc.

**Reply by sf202303** [reply]
Thanks so much for your help! One more question, where can I learn more about how to adjust the fragment length and/or what are sequencing/fragment parameters to be considered before metaSPAdes assembly?

**Reply by asl** [reply]
Well, SPAdes is not tuned to a particular properties of the input data, it tries to be robust and estimate from the input fas much as possible.

**Reply by sf202303** [reply]
Thanks again for all your replies and help!
